src/chrome/content/rules/tagesschau.de.xml

```diff
@@ -1,52 +1,43 @@
 <!--
 	Non-functional subdomains:
-		- atlas		(connection refused)
-
-	¹ Hostname mismatch
-
-	Insecure cookies are set for these domains:
-
-		- www.tagesschau.de
+		atlas (connection refused)
 
+	Redirect to Flypsite.com
+		duell.live.tagesschau.de
+		
+	Redirect to tagesschau.de
+		tagesschau24.de
+		www.tagesschau24.de
 
 	Mixed content:
-
-		- Images, on:
-
-			- wetter and www from wetter *
-			- wetter from www *
-
-		- Bugs, on:
-
-			- wetter and www from de.sitestat.com *
-			- wetter and www from gsea.ivwbox.de *
-			- wetter and www from tagessch.ivwbox.de *
-			- www from ts.flyp.tv *
-
-	* Secured by us
+		karten.tagesschau.de
+		wahl.tagesschau.de
```

Comment:
I was not able to reproduce mixed content issues on those hosts.

Reply:
My Firefox and Google Chrome give me this feedback with the mixed content.
Or see here:
wahl.tagesschau.de https://www.whynopadlock.com/results/9cf16086-e537-4f47-a650-df357f9fbc8b
karten.tagesschau.de https://www.whynopadlock.com/results/5efaad71-cf13-4b5f-9272-d8318090019d

Reply:
These are trackers, you can safely ignore them.